### ISSUE TITLE
Arkinventory not working with ElvUI

### ISSUE THREAD

**Issue by SlowmoFett** [arkayenro/arkinventory#1406]
I've been using Ark and ElvUI together for years but today Ark will not work with Elv loaded.  If I don't load Elv, Ark works fine.  I've disabled everything bag related in Elv and still can't get Ark to open, it only opens the standard blizzard bags.  Any help would be appreciated.

**Comment by SlowmoFett** [comment]
ok, I just found that if I do /arkinventory edit to get into that mode it will open and when I disable edit mode I can use stuff in my bags, however just opening my bags still opens the default blizzard bags.

**Comment by arkayenro** [comment]
what happens when you talk to a merchant?  do you get the ai bag or the default?

if elvui has done something odd when hooking the openbag functions then theres not a lot i can do - you could rebind the B key from openallbags to the arkinventory bag open instead

**Comment by arkayenro** [comment]
also, do you have other elvui mods loaded?  ie it might not be the base elvui it would be one one the other addons for it

**Comment by Alorel** [comment]
I'm also using ark with elvui and haven't had any issues - just had to disable elvui's bag manager in the settings. I think there might be another addon interfering

**Comment by SlowmoFett** [comment]
I've gone through and turned off all addons and loaded them one by one.  The only one that keeps Arkinventory from working is Elvui.  I've got the bag manager in elvui turned off but it still blocks Ark and I've figured out it keeps me from being able to open the config menu in Ark also.  I have no idea why it would do that.  

When I talk to a vendor it opens the default blizzard bags and not the Ark bags.
